### DISCUSSION TITLE
Order Priority

### DISCUSSION THREAD

**Discussion by naveed-alam** [woocommerce/woocommerce#44995] [WooCommerce, Blocks, and Site Editing]
Is there any possibility for the reposition the block inside the shipping/billing section. Like if someone want to add new block before country or after, is it possible?

Thank you.

**Comment by opr** [comment]
Hi, thanks for your question! Are you talking about inserting an inner block into the address fields, or just adding a new field?

Currently, it's not possible to do either, but we're working on allowing custom checkout fields to be added. woocommerce/woocommerce#42995 is the discussion for this effort.

If you are talking about inner blocks that aren't just regular form fields (input, select etc.) could you expand on your use case and give us some examples/screenshots to help us understand?

Thanks!

**Comment by naveed-alam** [comment]
Hi,  Thank you for your response.

1. It's possible to add inner block in shipping or billing fields, but I
want to add an input field (inner block) after billing country field.
2. Also is it possible to change address fields of billing address? What I
want to do is choose google suggested address and put them inside address
fields.

Thank you.
